metrics/METRICS.md

```diff
@@ -0,0 +1,77 @@
+
+# Deploying on OpenShift
+
+## Prometheus
+
+The Prometheus server configuration uses a service discovery feature in order to discover the pods in the cluster from which getting metrics.
+In order to have this feature working, it's needed to provide access to the API server for getting pods list to the service account used for running the Prometheus service pod.
+
+```
+export NAMESPACE=myproject
+oc login -u system:admin
+oc adm policy add-cluster-role-to-user cluster-reader system:serviceaccount:${NAMESPACE}:default
+oc login -u developer
+```
+
+After that :
+
+1. Create the [provided "prometheus" template](prometheus/openshift-template.yaml) by running
+
+        oc create -f metrics/prometheus/openshift-template.yaml
+
+2. Create a new app using the "prometheus" template:
+
+        oc new-app prometheus
+
+## Grafana
+
+For deploying the Grafana server in order to get a UI for the Prometheus metrics, you can use the following [project](https://github.com/OpenShiftDemos/grafana-openshift) which you need to clone locally.
+
+From the `grafana-openshift` directory, run
+
+```
+chmod +x root/usr/bin/fix-permissions 
+chmod +x run.sh 
+```
+for setting "execute" permissions on the above needed scripts and then run
+
+```
+oc new-build --binary --name=grafana
+oc start-build grafana --from-dir=. --follow
+```
+
+for starting the building image process inside the OpenShift cluster. 
+After a while (the first time), the grafana image will be available in the internal Docker registry of the OpenShift cluster and the last step is just creating a new application with the provided template.
+
+```
+oc new-app grafana
+```
+
+# Deploying on Kubernetes
+
+Create the Prometheus service by running
```

Comment:
For symmetry, this should probably be in a `## Prometheus` subsection

Reply:
You are absolutely right ! Added it.